src/map-app/app/view/sidebar/initiatives.ts

```diff
@@ -56,25 +59,32 @@ export class InitiativesSidebarView extends BaseSidebarView {
     d3.select("#search-box").property("value", txt);
   }
 
-  createSearchBox(selection: d3DivSelection) {
+  submitCallback = (event: Event) => {
```

Comment:
Looks like this declaration lost its `const` qualifier, becoming a global?

Reply:
I pulled it out of the scope of the `createSearchBox` method because I thought I needed to use it elsewhere in the class, but I didn't actually need to. I'll add it back.